Shifty.App/DomainModels/UnusedTicket.cs

```diff
@@ -0,0 +1,24 @@
+using Components;
+using Shifty.Api.Generated.AnalogCoreV2;
+
+namespace Shifty.App.DomainModels
+{
+    public class UnusedTicket
+    {
+        public int ProductId { get; set; }
+        public string ProductName { get; set; }
+        public int TicketsLeft { get; set; }
+        public decimal UnusedPurchasesValue { get; set; }
+        
+        public static UnusedTicket FromDto(UnusedClipsResponse ticket)
```

Comment:
We probably want to look into a mapping utility, e.g. mapperly or mapster, but doesn't need to be part of this pr.